src/cdk/tree/tree.ts

```diff
@@ -331,23 +340,30 @@ export class CdkTreeNode<T> implements FocusableOption, OnDestroy {
   }
   protected _data: T;
 
-  get isExpanded(): boolean {
+  // tslint:disable-next-line:no-host-decorator-in-concrete
+  @HostBinding('attr.aria-expanded') get isExpanded(): boolean {
```

Comment:
Won't this have the same issue as the `role` host binding if you don't set it via `setAttribute`?